lib/node-helpers.js

```diff
@@ -18,6 +41,18 @@ module.exports = {
   },
 
   /**
+   * Tells if the current process has freed
+   * it s resources and thus is ready to quit properly
+   */
+  hasFreedResources: function () {
+    /*eslint-disable */
+    var handlesC = process._getActiveHandles().length;
+    var requestsC = process._getActiveRequests().length;
+    /*eslint-enable */
+    return handlesC + requestsC === 0;
```

Comment:
I don't know how js evaluate operators but sometime that kind of writing could be interpreted as

```javascript
return handlesC + (requestsC === 0);
```
A safer way to write it, would be:

```javascript
return (handlesC + requestsC) === 0;
```